mvcc/backend/batch_tx.go

```diff
@@ -165,7 +166,9 @@ func (t *batchTx) commit(stop bool) {
 			atomic.StoreInt64(&t.backend.size, t.tx.Size())
 			return
 		}
+		start := time.Now()
 		err = t.tx.Commit()
+		commitDurations.Observe(time.Since(start).Seconds())
```

Comment:
or `Observe(float64(time.Since(start)) / float64(time.Second))` like in `syncDurations`